app/views/layouts/application.html.erb

```diff
@@ -15,6 +15,7 @@
     <%= render "shared/flashes" %>
 
     <%= yield %>
+    <%= render "shared/topbar" %>
```

Comment:
In all logic this should be **above** the rest of the content, so above the body